.environment/gitleaks/gitleaks-config.toml

```diff
@@ -62,6 +64,7 @@ title = "PRIME ReportStream Gitleaks Configuration"
             '(?i)joe.jones@az.pima.gov',
             '(?i)local@test.com',
             '(?i)noreply@cdc.gov',
+            '(?i)no-reply@cdc.gov',
```

Comment:
I had to add this after I merged in master. I'm not sure how this got into master but broke my gitleaks check.